src/components/footer/Footer.jsx

```diff
@@ -0,0 +1,34 @@
+import "./footer.css";
+
+const Footer = () => {
+  return (
+    <footer className="footer">
+      <p>Made by Yaswanth Myneni</p>
+      <div className="footer-flex">
+        <a
+          target="_blank"
+          href="https://github.com/yaswanthmyneni/"
+          rel="noreferrer"
+        >
+          <i className="fa-brands fa-github social-links"></i>
+        </a>
```

Comment:
Suggestion: You can use react icons library for icons it would be good and easy to integrate here